packages/picongpu/package.py

```diff
@@ -3,7 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 #
-# Authors: Axel Huebl
+# Authors: Axel Huebl, Rene Widera
```

Comment:
Fantastic, can you also add yourself to the `maintainers` member list please? :)

Reply:
added me to the maintainer list